BackEndApp/sample.env

```diff
@@ -1,6 +1,15 @@
+# required
 MONGODB_URI=mongodb://localhost:27017/
 JWT_SECRET=abcdefh12345
+
+# model deployments
+FLASK_APP_HTTPS_URL=https://github.com/Auto-DL/Deployment-Flask.git
+FLASK_APP_SSH_URL=git@github.com:Auto-DL/Deployment-Flask.git
+
+# client app url
```

Comment:
I think we can move this to required itself? 🤔  or add `(required)` in front of it..

Reply:
I've moved it to required itself 👍🏼